### PR TITLE
add Request Line

### PR DESCRIPTION
**Pull request by Sammers21**
Request line would simplify request line formation by a server

### CODE REVIEW

File: src/main/java/com/artipie/http/RequestLine.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+     */
+    public RequestLine(final String method, final String uri, final String version) {
+        this.method = method;
+        this.uri = uri;
+        this.version = version;
+    }
+
+    @Override
+    public String toString() {
+        return String.format("%s %s %s\r\n", this.method, this.uri, this.version);
```
Comment (g4s8) [inline src/main/java/com/artipie/http/RequestLine.java]: @Sammers21 we don't need these line-endings here `\r\n`, it's the separator for request line and headers

Comment (Sammers21) [reply src/main/java/com/artipie/http/RequestLine.java]: according to spec: `Request-Line   = Method SP Request-URI SP HTTP-Version CRLF`

Comment (g4s8) [reply src/main/java/com/artipie/http/RequestLine.java]: @Sammers21 updated in https://github.com/artipie/http/pull/13/commits/6e1d00a2638378a143df47aeae7854acf65688ec

**Comment by yegor256** [comment]
@Sammers21 no unit test?

**Comment by Sammers21** [comment]
@yegor256, just created a test.

**Comment by 0crat** [comment]
This job is not in scope

**Comment by 0crat** [comment]
Job `gh:artipie/http#23` is not assigned, can't get performer

**Pull request merged**